okhttp/src/jvmMain/kotlin/okhttp3/internal/connection/RealConnectionPool.kt

```diff
@@ -252,7 +265,6 @@ class RealConnectionPool(
       Platform.get().logCloseableLeak(message, callReference.callStackTrace)
 
       references.removeAt(i)
-      connection.noNewExchanges = true
```

Comment:
Very small behavior change here - if we leak a Call that doesn‘t immediately mark the connection with noNewExchanges. That’s fine.